types/video.js/video.js-tests.ts

```diff
@@ -159,6 +159,12 @@ videojs('example_video_1', playerOptions).ready(function playerReady() {
 
     const readyState: videojs.ReadyState = this.readyState();
 
+    // $ExpectType string
+    const currentBreakPoint: string = this.currentBreakpoint();
```

Comment:
```suggestion
    const currentBreakPoint = this.currentBreakpoint();
```
(as per assertion, type is infered)